scripts/setup_auto_sourcing.bash

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
the script was not working when I already have had .ros_team_ws file in my workspace. why not back it up in `.ros_team_ws.bkp` file and create a new one. And also notify a user about it.

Reply:
Can you also please use colors when notifying user? Depending on what you want, confirmation of show information then we should use different colors. They are already defined in `_RosTeamWs_Defines` file.

Reply:
Done

Reply:
Did also fix the second (not here but in nextcloud mntioned) error with this [commit](https://github.com/StoglRobotics/ros_team_workspace/pull/55/commits/1baa8d1797150ebf416769bf7af42f176a5414bf)